client/src/pages/OrganizerEvent/OrganizerEvent.js

```diff
@@ -9,13 +9,17 @@ import "./style.css";
 import { TeamOutlined } from "@ant-design/icons";
 import ModalPopUp from "../../components/ModalPopUp/ModalPopUp";
 import { useAuth0 } from "@auth0/auth0-react";
-import { createParticipantLogic } from "../../actions/graphql.api";
+import {
+	createParticipantLogic,
+	getParticipantsbyEventId,
+} from "../../actions/graphql.api";
 
 // Notification for when participant is entered successfully
 const showSuccess = () => {
 	notification.success({
 		message: "Participant Added",
-		description: "Participant successfully added.",
+		description:
+			"Participant successfully added. Refresh to see new participants.",
```

Comment:
I think instead of asking them to refresh the page, we should automatically reload participants again.